apps/client/src/features/control/message/MessageControl.tsx

```diff
@@ -42,20 +41,20 @@ export default function MessageControl() {
       <div className={style.buttonSection}>
         <Button
           size='sm'
-          className={`${data.timerMessage.timerBlink ? style.blink : ''}`}
-          variant={data.timerMessage.timerBlink ? 'ontime-filled' : 'ontime-subtle'}
-          leftIcon={data.timerMessage.timerBlink ? <IoSunny size='1rem' /> : <IoSunnyOutline size='1rem' />}
-          onClick={() => setMessage.timerBlink(!data.timerMessage.timerBlink)}
+          className={`${data.timerMessage.blink ? style.blink : ''}`}
+          variant={data.timerMessage.blink ? 'ontime-filled' : 'ontime-subtle'}
+          leftIcon={data.timerMessage.blink ? <IoSunny size='1rem' /> : <IoSunnyOutline size='1rem' />}
+          onClick={() => setMessage.timerBlink(!data.timerMessage.blink)}
```

Comment:
Would it be helpful to extract this into a variable at the top like `const blink = data.timerMessage.blink`?
This is a little verbose....

Same for blackout

Reply:
do I need to put it in a memo? or something like that?

Reply:
No. We are just creating a label for a value. No computations here 

Memo should be reserved for expensive computations or computations which trigger invalidation of data or potentially expensive components